<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>901</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网络数据包发送工具</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="procotolSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>协议选择</string>
   </property>
   <widget class="QRadioButton" name="EtherradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ARPradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IPradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="UDPradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TCPradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="parameterSet">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Ether">
    <widget class="QLabel" name="EtherdstMAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目的MAC地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Etherpayload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>负载数据：</string>
     </property>
    </widget>
    <widget class="QLabel" name="EthersendCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送次数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EtherdstMACEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EtherpayloadEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EthersendCountEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="EthersrcMAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>源MAC地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EthersrcMACEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="IP">
    <widget class="QLabel" name="IPdstIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IP目的IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="IPsrcIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IP源IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="IPpayload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>负载数据：</string>
     </property>
    </widget>
    <widget class="QLabel" name="IPsendCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送次数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPdstIPEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPsrcIPEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPpayloadEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPsendCountEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UDP">
    <widget class="QLabel" name="UDPdstIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UDP目的IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDPsrcIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UDP源IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDPdstPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UDP目的端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDPsrcPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UDP源端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDPpayload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>负载数据：</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDPsendCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送次数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPdstIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPsrcIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPdstPortEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPsrcPortEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPpayloadEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UDPsendCountEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TCP">
    <widget class="QLabel" name="TCPdstIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TCP目的IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TCPsrcIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TCP源IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TCPdstPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TCP目的端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TCPsrcPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TCP源端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TCPpayload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>负载数据：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TCPsendCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送次数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPdstIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPsrcIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPdstPortEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPsrcPortEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPpayloadEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TCPsendCountEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ARP">
    <widget class="QLabel" name="dstMAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目的MAC：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARPdstMAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ARP目的MAC：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARPdstIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ARP目的IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARPsrcMAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ARP源MAC：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARPsrcIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ARP源IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARPpayload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>负载数据：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dstMACEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPdstMACEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPdstIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPsrcMACEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPsrcIPEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPpayloadEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ARPsendCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送次数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ARPsendCountEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="sendButtonSet">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="EtherSend">
    <widget class="QPushButton" name="EtherSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送数据包</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="IPSend">
    <widget class="QPushButton" name="IPSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送数据包</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UDPSend">
    <widget class="QPushButton" name="UDPSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送数据包</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TCPSend">
    <widget class="QPushButton" name="TCPSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送数据包</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ARPSend">
    <widget class="QPushButton" name="ARPSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送数据包</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sendResult">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送结果</string>
   </property>
   <widget class="QTextEdit" name="sendResultEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sendHistory">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>561</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送历史</string>
   </property>
   <widget class="QTableWidget" name="sendHistoryWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>个数</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
